### PR TITLE
chore: 🤖 improve cli-flags usage

### PR DESCRIPTION
**Pull request by snitin315**
**What kind of change does this PR introduce?**
Chore

**Did you add tests for your changes?**
NA
**If relevant, did you update the documentation?**
NA
**Summary**
improved flags example usage for some String type flags.

**Does this PR introduce a breaking change?**
No

**Other information**

### CODE REVIEW

File: packages/webpack-cli/lib/utils/cli-flags.js [hunk 1]
```diff
@@ -121,28 +121,28 @@ module.exports = {
...
             link: 'https://webpack.js.org/concepts/#output',
         },
         {
             name: 'plugin',
-            usage: '--plugin',
+            usage: '--plugin <pluginName>',
             group: ADVANCED_GROUP,
             type: String,
-            description: 'Load a given plugin',
+            description: 'Load a given plugin e.g. HtmlWebpackPlugin',
```
Comment (anshumanv) [inline packages/webpack-cli/lib/utils/cli-flags.js:136]: This is only to generate plugin iirc?

Comment (jamesgeorge007) [reply packages/webpack-cli/lib/utils/cli-flags.js:136]: I assume you're speaking about the `plugin` command which is used to scaffold a plugin.
https://github.com/webpack/webpack-cli/blob/49620e0070a1be9287b493fe22bfcfab650341f3/packages/webpack-cli/lib/utils/cli-flags.js#L42

Comment (jamesgeorge007) [reply packages/webpack-cli/lib/utils/cli-flags.js:136]: It isn't the case here.

Comment (anshumanv) [reply packages/webpack-cli/lib/utils/cli-flags.js:136]: Ah okay yes

**Pull request merged**